src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function dummyMod(m)
```

Comment:
which version do you use? maybe time for an update!? Here is the documentation:  http://docs.julialang.org/en/release-0.4/manual/linear-algebra/